Sources/SwiftSyntaxBuilder/FunctionCallExprConvenienceInitializers.swift

```diff
@@ -16,19 +16,33 @@ import SwiftSyntax
 extension FunctionCallExpr {
   public init(
     _ calledExpression: ExpressibleAsExprBuildable,
-    leftParen: TokenSyntax? = .leftParen,
-    rightParen: TokenSyntax? = .rightParen,
+    leftParen: TokenSyntax? = nil,
+    rightParen: TokenSyntax? = nil,
     trailingClosure: ExpressibleAsClosureExpr? = nil,
     @TupleExprElementListBuilder argumentListBuilder: () -> ExpressibleAsTupleExprElementList = { TupleExprElementList([]) },
     @MultipleTrailingClosureElementListBuilder additionalTrailingClosuresBuilder: () -> MultipleTrailingClosureElementList? = { nil }
   ) {
+    let additionalTrailingClosures = additionalTrailingClosuresBuilder()
+    let argumentList = argumentListBuilder().createTupleExprElementList()
+    let shouldSetParens = argumentList.elements.isEmpty == false
+      || (trailingClosure == nil && additionalTrailingClosures?.elements.isEmpty != false)
+    var leftParen = leftParen
+    if leftParen == nil && shouldSetParens {
```

Comment:
Nitpick: The preferred spelling in Swift would be

```suggestion
    if leftParen == nil, shouldSetParens {
```